dsl/src/main/scala/com.crobox.clickhouse/dsl/OperationalQuery.scala

```diff
@@ -34,18 +32,17 @@ trait OperationalQuery extends Query {
     OperationalQuery(internalQuery.copy(where = Some(comparison)))
   }
 
-  def from[T <: Table](table: T): OperationalQuery = {
-    val from = TableFromQuery(table)
-    OperationalQuery(internalQuery.copy(from = Some(from)))
-  }
+  def as(alias: String): OperationalQuery =
+    OperationalQuery(internalQuery.copy(as = Some(alias)))
 
-  def from(query: OperationalQuery): OperationalQuery = {
-    val from = InnerFromQuery(query)
-    OperationalQuery(internalQuery.copy(from = Some(from)))
-  }
+  def from[T <: Table](table: T): OperationalQuery =
+    OperationalQuery(internalQuery.copy(from = Some(TableFromQuery(table))))
+
+  def from(query: OperationalQuery): OperationalQuery =
+    OperationalQuery(internalQuery.copy(from = Some(InnerFromQuery(query))))
 
   def asFinal: OperationalQuery =
-    OperationalQuery(internalQuery.copy(asFinal = true))
+    OperationalQuery(internalQuery.copy(as = Option("FINAL")))
```

Comment:
FINAL is not the same as AS (aliasing) right? How do i now do `SELECT * FROM table FINAL AS other`?